cookbooks/aws-parallelcluster-config/resources/manage_ebs.rb

```diff
@@ -90,8 +90,9 @@
       options "_netdev"
       pass 0
       action %i(mount enable)
-      retries 10
+      retries 1
       retry_delay 6
+      not_if "mountpoint -q #{shared_dir_array[index]}"
```

Comment:
Minor (let's consider it outside of this PR): we reference `shared_dir_array[index]` in many places; I think we should use a more explicit variable like `mount_point = shared_dir_array[index]`

Reply:
since this PR is high priority, let's address all the clean coding comments in another PR. I'll take care of it